back/Gemfile

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
+ruby file: '.ruby-version'
```

Comment:
Would it be simpler to just specify it like `ruby '2.7.6'` and remove the file? This way, we don't need to copy this file to the dockerfiles. Or we use it for something else too?

Reply:
This file is also used by other tools. For instance, I think `rbenv` picks it up and automatically uses the ruby version specified in the file.